### DISCUSSION TITLE
승객 및 열차

### DISCUSSION THREAD

**Discussion by seong0929** [Tiny-Metro/Poly-Metro#53] [Ideas]
- 승객 색상
- 승객 모형
- 등장과 퇴장씬
- 탑승씬

**Comment by seong0929** [comment]
![image](https://user-images.githubusercontent.com/50050003/208240962-a5d06cb6-a126-4536-93fb-1ccaa82f6de7.png)

- 열차 모양: 슬롯 형태 모양
- 승객 색상: 해당 역의 불만도와 비례하게 색깔이 바뀌도록(불만도가 낮을때(25): 파랑, 중간일때(50): 노랑, 높을 때(75)는 빨강)
- 승객 모형: 가고자 하는 역의 모양이 서 있는 카드 모양
- 열차 탑승 시: 45도 기울어진 모양
- 등장과 탑승, 탑승은 순간 이동씬
- 대기: 역 오른쪽으로 2열 횡대 정렬

**Reply by cstrnull00** [reply]
기본 동차의 승차 인원은 6명(2열 3행), 업그레이드 시 10명(2열 5행), 색은 노선 색에 따라감
객차의 승차 인원, 업그레이드도 같음.
